src/main/kotlin/com/atlassian/performance/tools/dockerinfrastructure/api/jira/AbstractJiraFormula.kt

```diff
@@ -38,15 +38,14 @@ abstract class AbstractJiraFormula internal constructor(
             ImageFromDockerfile()
                 .withDockerfileFromBuilder { builder ->
                     builder
-                        .from("ubuntu:20.04")
+                        .from("amd64/ubuntu:20.04")
```

Comment:
No need for this, when we remove JAVA_HOME

Reply:
Commited by mistake when testing forced architecture on arm. It takes ~100% more time to finish the test